rlp/rlpgen/gen.go

```diff
@@ -158,7 +158,7 @@ type op interface {
 // basicOp handles basic types bool, uint*, string.
 type basicOp struct {
 	typ           types.Type
-	writeMethod   string     // calle write the value
+	writeMethod   string     // call write the value
```

Comment:
@fjl What should be here?

Reply:
maybe "EncoderBuffer writer method name"